.tool-versions

```diff
@@ -4,6 +4,7 @@ pre-commit 3.6.0
 terraform 1.9.2
 vale 3.6.0
 tfsec 1.28.10
+nodejs 20.13.1
```

Comment:
Do we also need to make sure that the backend lambda runtimes stays in sync with this?

Reply:
updated